Duplicate recipe in the Makefile

There are a couple recipes which are duplicated in the Makefile

```
Makefile:110: warning: overriding recipe for target 'visualization'
Makefile:95: warning: ignoring old recipe for target 'visualization'
Makefile:113: warning: overriding recipe for target 'admin'
Makefile:20: warning: ignoring old recipe for target 'admin'
````

--- a/Makefile
+++ b/Makefile
@@ -106,12 +106,6 @@ frontend: oc_login
 leaderboard: oc_login
 	./leaderboard/deploy.sh
 
-visualization: oc_login
-	./visualization/deploy.sh
-
-admin: oc_login
-	./admin-hq/deploy.sh
-
 ml: oc_login
 	./digit-recognition/deploy.sh
 

